tests/parm/nems.configure.cpld_outwav.IN

```diff
@@ -0,0 +1,141 @@
+#############################################
```

Comment:
I don't see any test using this nems.configure file. Is there are reason it is needed?

Reply:
We're just adding the one test for now, but it's very likely that we'll use this configuration in EP4, so the other coupled cases, so I was being proactive and including it now.  It can be removed for the moment.

Reply:
So why is the added test a 'noaero' test and not just an outer-loop version of cpld_control_p8?

Reply:
Because for now we wanted to keep the testing burden low and we know for sure we will use this for GFSv17, when we know for sure this will also be for GEFS, we can update the p8 tests as well.